src/app/components/add-recipe-page/add-recipe-page.component.ts

```diff
@@ -64,20 +67,21 @@ export class AddRecipePageComponent implements OnInit {
     return this.recipeForm.get('preparingSteps') as FormArray;
   }
 
-  get f() { return this.recipeForm.controls; }
+  get f() {
```

Comment:
What is `f`? Don't use single letter variables (there are some exception but this is not one of them)